libtransmission/torrent.cc

```diff
@@ -2568,8 +2568,8 @@ void renameTorrentFileString(tr_torrent* tor, std::string_view oldpath, std::str
 
 void torrentRenamePath(
     tr_torrent* const tor,
-    std::string oldpath, // NOLINT performance-unnecessary-value-param
-    std::string newname, // NOLINT performance-unnecessary-value-param
+    std::string const& oldpath, // NOLINT performance-unnecessary-value-param
```

Comment:
I'm not familiar with NOLINT annotations, but if you're fixing the performance, do we still need this annotation?